Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,28 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testEncodePlist() {
+        let point = CGPoint(x: 1.1, y: 2.2)
+        let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+        let temporaryFilename = ProcessInfo().globallyUniqueString
+        let fileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+        do {
+            try FileManager.default.encodePlist(point, to: fileURL)
+
+            XCTAssert(FileManager.default.fileExists(atPath: fileURL.path))
+
+            let data = FileManager.default.contents(atPath: fileURL.path)
+            XCTAssertNotNil(data)
+
+            let decoder = PropertyListDecoder()
+            let decodedPoint = try decoder.decode(CGPoint.self, from: data!)
+
+            XCTAssert(decodedPoint == point)
```

Comment:
Use `XCTAssertEqual`
```suggestion
            XCTAssertEqual(decodedPoint, point)
```